pandas/core/generic.py

```diff
@@ -3156,6 +3156,13 @@ def copy(self, deep=True):
         data = self._data.copy(deep=deep)
         return self._constructor(data).__finalize__(self)
 
+    __copy__ = copy
```

Comment:
Your prior way seems more elegant (and faster)! Unfortunately Python fails the elegant = correct question here, though